config/routes.rb

```diff
@@ -2,6 +2,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   resources :countries, only: [:show]
+  get '/travel/:country_1/:country_2', to: 'countries#render', as: 'countries_render'
```

Comment:
@iandouglas @AliSchlereth Is this the best way to handle this? Haven't build out `CountriesController#render` yet, but if I'm trying to allow access to an endpoint such as `/travel/US/GB` (travel from US to England), is that the proper way to go about this? Or should I make it using nested resources, so it's more like `/country/US/country/GB`. The latter feels nice from a UX standpoint.

Reply:
I want to think about the uri a bit more, but Do have a comment about using render as an action. Try to stick to the 7 restful actions in your controller. Try to match what you're doing to whichever action fits best, also remember you can create another controller if you already have used that action in the CountriesController.